.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @Azure/azure-osconfig-core @MariusNi @robertschaedler3
```

Comment:
>@MariusNi [](http://example.com/codeflow?start=29&length=9)

Put MariusNi back please

Reply:
As described below, this will not change the notification behavior or PR requirements.

Reply:
As discussed, we shall not change the team names and code owners for adding more people to be code reviewers. Recommend to leave all else as-is and add the new members to azure-osconfig-core (existing) and azure-osconfig-extended (new). Leave codeowners set to azure-core-osconfig *plus* at minimum MariusNi.